tests/zfs-tests/tests/functional/channel_program/lua_core/setup.ksh

```diff
@@ -18,4 +18,10 @@
 
 DISK=${DISKS%% *}
 
+if [[ "$(uname -m)" == "aarch64" ]] && is_linux ; then
+	if awk '/^CPU implementer/ {exit !/0x61/}' /proc/cpuinfo; then
+		log_unsupported "Not supported on Apple Silicon"
```

Comment:
Probably wouldn't hurt to be specific:
"Not supported on Linux VMs running on Apple ARM silicon"

Reply:
So the tests may also not work on FreeBSD, but I wasn't sure how to check if a VM is running on Apple SIlicon on FreeBSD.

Reply:
:+1: